dashboard/client/src/pages/node/index.tsx

```diff
@@ -283,7 +283,7 @@ const Nodes = () => {
                       key={node.raylet.nodeId}
                       node={node}
                       isRefreshing={isRefreshing}
-                      startExpanded={i === 0}
+                      startExpanded={nodeList.length === 1 && i === 0}
```

Comment:
nit: The `i == 0` is redundant / never evaluated.

Reply:
:)